l10n/en/firefox/sticky-promo.ftl

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/*
```

Comment:
I think this field pops up in Pontoon for translators to click and preview their work. Suggest just putting one URL in here where the promo appears?
```suggestion
### URL: https://www-dev.allizom.org/firefox/
```

Reply:
Revised the URL and ammended the line below about the scope of this text.